core/consensus/common/primitives/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "substrate-consensus-common-primitives"
+version = "2.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+description = "Primitives for session"
```

Comment:
Copy pasta?

Reply:
Good catch. I was implementing this as a runtime api against the session module first.